config/jobs/kubernetes-sigs/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -276,7 +276,7 @@ presubmits:
     extra_refs:
       - org: kubernetes-sigs
         repo: cluster-api-provider-azure
-        base_ref: release-1.7
+        base_ref: release-1.8
```

Comment:
Unsuccessful test run after this change:

- https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/directory/pull-cloud-provider-azure-e2e-ccm-vmss-ip-lb-capz/1634187421539110912

Does not appear to be CAPZ related, but FYI